CHANGELOG.md

```diff
@@ -1,7 +1,20 @@
 # Changelog
 
 ## [0.13.2](https://github.com/brave/browser-laptop/releases/v0.13.2dev)
-- TODO
+- Add muon and vault-updater support for Enpass. ([#6906](https://github.com/brave/browser-laptop/issues/6906))
+- Fixed spellcheck on Linux. ([#7015](https://github.com/brave/browser-laptop/issues/7015))([#6967](https://github.com/brave/browser-laptop/issues/6967))
+- Fixed Command+Click to open a new tab. ([#6971](https://github.com/brave/browser-laptop/issues/6971))
+- Fixed various UI issues. ([#6812](https://github.com/brave/browser-laptop/issues/6812))([#6806](https://github.com/brave/browser-laptop/issues/6806))([#6867](https://github.com/brave/browser-laptop/issues/6867))([#6754](https://github.com/brave/browser-laptop/issues/6754))
+- Fixed various ledger issues. ([#6966](https://github.com/brave/browser-laptop/issues/6966))([#6592](https://github.com/brave/browser-laptop/issues/6592))([#6878](https://github.com/brave/browser-laptop/issues/6878))([#6833](https://github.com/brave/browser-laptop/issues/6833))([#6685](https://github.com/brave/browser-laptop/issues/6685))([#6941](https://github.com/brave/browser-laptop/issues/6941))
```

Comment:
Are these included? https://github.com/brave/browser-laptop/issues?utf8=%E2%9C%93&q=is%3Aclosed%20is%3Aissue%20milestone%3A0.13.2%20label%3Afeature%2Fledger%20